lib/usb/usb.c

```diff
@@ -94,6 +96,19 @@ void usbd_register_sof_callback(usbd_device *usbd_dev, void (*callback)(void))
 	usbd_dev->user_callback_sof = callback;
 }
 
+void usbd_register_extra_string(usbd_device *usbd_dev, int index, const char* string)
+{
+	if (string != NULL && index > 0)
```

Comment:
given this, what's wrong with just using zero for the default of extra_string_idx?